src/test/javascript/stringExtensionSpec.js

```diff
@@ -56,6 +56,14 @@ define(['bytepushers'], function(BytePushers) {
                 // Hello this is a sentence => helloThisIsASentence => Hello This Is ASentence
                 expect(result).not.toBe("Hello bytePushers");
             });
+            it('can format a sentence into camel case with a single letter word', function () {
+                var result = "Hello captainfalcon".toCamelCase().toNormalCase();
+                // Would think it would do this
+                // Hello this is a sentence => helloThisIsASentence => Hello This Is A Sentence
+                // Actually does this
+                // Hello this is a sentence => helloThisIsASentence => Hello This Is ASentence
```

Comment:
placement of comments could be minimized to one line if possible and no space between var and expect.
```suggestion
                // Hello this is a sentence => helloThisIsASentence => Hello This Is ASentence
```

Reply:
Left comments on initial it test and removed comments on following it tests to clean up code.